problem_builder/dashboard.py

```diff
@@ -412,6 +414,10 @@ def student_view(self, context=None):  # pylint: disable=unused-argument
             if numeric_values:
                 average_value = sum(numeric_values) / len(numeric_values)
                 block['average'] = average_value
+                # average block is shown only if average value exists, so accessible text for no data is not required
+                block['accessible_average'] = _("Score: {average_value}").format(
```

Comment:
Just a note about translations: this will be in your po file as a _separate translation string_ from the `"Score: {value}"` string above. However... why? They're the same exact string, semantically. If you match them exactly, translators will only have to translate the strings once.